detox/e2e/support/server_api/system.ts

```
@@ -413,6 +442,7 @@ export const System = {
     apiGetRemoteClusters,
     apiPatchConfig,
     apiPingServerStatus,
+    // apiRequestTrialLicense, // DISABLED: Do not request trial license in tests
```

ESLint

[error] Expected line before comment.

(lines-around-comment)